recipe/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "scispacy" %}
+{% set version = "0.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scispacy-{{ version }}.tar.gz
+  sha256: 12bd915a81712a6dd50284ee7518088a8dd2c75d69949ea1bd3df00aee763d69
+
+build:
+  # spacy is not available on s390x
+  skip: true  # [py<36 or (linux and s390x)]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - wheel
+  run:
+    - python 
+    - spacy >=3.4.0,<3.5.0
+    - requests >=2.0.0,<3.0.0
+    - conllu
+    - numpy
+    - joblib
+    - nmslib >=1.7.3.6
+    - scikit-learn >=0.20.3
+    - pysbd
```

Comment:
does it run without it ?, it says in upstream is for testing or  "custom sentence splitter" which I really dunno if it used or not

Reply:
Looking at the code it seems it uses it during runtime. Also, this scispacy is about processing text, so I expect this pysbd package to be used.